contentctl/objects/abstract_security_content_objects/detection_abstract.py

```diff
@@ -657,68 +737,82 @@ def search_observables_exist_validate(self):
-        # Accordingly, we do not need to do additional checks if the type is Correlation
-        if self.type in set([AnalyticsType.Correlation.value]):
-            return self
+    #     # Only production analytics require tests
+    #     if self.status != DetectionStatus.production.value:                                         # type: ignore
+    #         return self
 
-        if self.tags.manual_test is not None:
-            for test in self.tests:
-                test.skip(
-                    f"TEST SKIPPED: Detection marked as 'manual_test' with explanation: '{self.tags.manual_test}'"
-                )
+    #     # All types EXCEPT Correlation MUST have test(s). Any other type, including newly defined types, requires them.
+    #     # Accordingly, we do not need to do additional checks if the type is Correlation
+    #     if self.type in UNTESTED_ANALYTICS_TYPES:
+    #         return self
 
-        if len(self.tests) == 0:
-            raise ValueError(f"At least one test is REQUIRED for production detection: {self.name}")
+    #     if len(self.tests) == 0:
+    #         raise ValueError(f"At least one test is REQUIRED for production detection: {self.name}")
 
-        return self
+    #     return self
 
     @field_validator("tests")
     def tests_validate(
         cls,
-        v: list[UnitTest | IntegrationTest],
+        v: list[UnitTest | IntegrationTest | ManualTest],
         info: ValidationInfo
-    ) -> list[UnitTest | IntegrationTest]:
+    ) -> list[UnitTest | IntegrationTest | ManualTest]:
         # Only production analytics require tests
         if info.data.get("status", "") != DetectionStatus.production.value:
             return v
 
         # All types EXCEPT Correlation MUST have test(s). Any other type, including newly defined types, requires them.
         # Accordingly, we do not need to do additional checks if the type is Correlation
-        if info.data.get("type", "") in set([AnalyticsType.Correlation.value]):
+        if info.data.get("type", "") in UNTESTED_ANALYTICS_TYPES:
+            return v
+
+        # Manually tested detections are not required to have tests defined
+        tags: DetectionTags | None = info.data.get("tags", None)
+        if tags is not None and tags.manual_test is not None:
             return v
 
         # Ensure that there is at least 1 test
         if len(v) == 0:
-            if info.data.get("tags", None) and info.data.get("tags").manual_test is not None:       # type: ignore
-                # Detections that are manual_test MAY have detections, but it is not required.  If they
-                # do not have one, then create one which will be a placeholder.
-                # Note that this fake UnitTest (and by extension, Integration Test) will NOT be generated
-                # if there ARE test(s) defined for a Detection.
-                placeholder_test = UnitTest(                                                        # type: ignore
```

Comment:
No longer need these placeholder tests; we allow manual test detections to have an empty list for tests as we do for non-prod detections

Reply:
Agreed!